src/Common/quoteString.h

```diff
@@ -16,12 +16,12 @@ namespace DB
 }
 
 /// Double quote the string.
-String doubleQuoteString(const StringRef & x);
+String doubleQuoteString(StringRef x);
 
 /// Quote the identifier with backquotes.
-String backQuote(const StringRef & x);
+String backQuote(StringRef x);
 
 /// Quote the identifier with backquotes, if required.
-String backQuoteIfNeed(const StringRef & x);
+String backQuoteIfNeed(StringRef x);
```

Comment:
Maybe it would be cool in the future to replace `StringRef` with `std::string_view` because both classes are almost the same.

Reply:
Yeah, I also thought about that ... shouldn't be too difficult.